tutorials/fiori-ios-scpms-starter-mission-02/fiori-ios-scpms-starter-mission-02.md

```diff
@@ -590,7 +590,7 @@ Before we do this we have to take care of the product image lazy loading. We're
 
 4. Before we can start dequeuing the needed cells, complete the `viewDidLoad()` method:
```

Comment:
Please don't say "we", say "you" instead as described in the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqgno1sd).